lib/taiko.js

```diff
@@ -3199,12 +3202,15 @@ const rectangle = async (selector, callback) => {
  * @property {function()} isDisabled - Checks if element is disabled and returns a boolean.
  * @property {string} description - Describes the operation performed.
  * @property {function} text - Gives innerText of the element found.
+ * @property {function} attribute - Gives attribute value of the element found.
  *
  * @example
  * link('google').exists()
  * @example
  * link('google').exists(1000)
  * @example
+ * $('.asdfb').attribute('name)
```

Comment:
@saikrishna321 can you change this to `link('google').attribute('alt')` assuming that this work. Even other wise there is an ending quote missing at `'name`